subscriber.rb

```diff
@@ -0,0 +1,48 @@
+
+require 'aws-sdk-sqs'
+require 'byebug'
+require_relative 'load_aws'
+
+class Subscriber
+  
+  ACCESS_KEY_ID = AWS["access_key_id"]
+  SECRET_ACCESS_KEY = AWS["secret_access_key"]
+  REGION = AWS["region"]
```

Comment:
This and above not needed anywhere else - except when creating AWS creds. Can be moved.

Reply:
> How do you run the code?

ruby subscriber.rb <queue_url(required)> <max-number-of-messages(optional)> <wait-time-seconds(optional)>

Reply:
> This and above not needed anywhere else - except when creating AWS creds. Can be moved.

We need the sqs object for which we would need to AWS constant. Its redundant, but considering subscriber as a stand-alone service, we would need them in this file.

Reply:
Meant that you don't need to have these as constant accessible throughout the code. Use it within the `new` method.

```
def initialize
    creds = Aws::Credentials.new(AWS["ACCESS_KEY_ID", AWS["SECRET_ACCESS_KEY"])
    @sqs = Aws::SQS::Client.new(region: AWS["REGION"], credentials: creds)
end
```